source/core/Variants.cpp

```diff
@@ -37,6 +37,7 @@ VariantType::VariantType(TypeManagerRef typeManager)
     _topAQSize = TypeManager::HostPointerToAQSize();
     _aqAlignment = TypeManager::AQAlignment(sizeof(void*));
     _encoding = kEncoding_Variant;
+    _isValid = 1;
```

Comment:
I think this change was just done by Sid.

Reply:
Rebased on top of Sid's changes in master